web/modules/custom/mymodule/src/Form/FirstForm.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Drupal\mymodule\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+class FirstForm extends FormBase {
```

Comment:
The question in the ticket mentions to create a config form.
Is this a config form?

Reply:
Alright.